src/app/window-handler.ts

```diff
@@ -329,14 +336,17 @@ export class WindowHandler {
                 logger.info(`window-handler: main window web contents destroyed already! exiting`);
                 return;
             }
+            this.finishedLoading = true;
             this.url = this.mainWindow.webContents.getURL();
+            if (this.url.indexOf('about:blank') === 0) {
+                logger.info(`Looks like about:blank got loaded which may lead to blank screen`);
+                logger.info(`Restarting app to check if it resolves the issue`);
+                app.relaunch();
```

Comment:
Instead of relaunching the application  can we just load the window with the URL? Will that not fix the issue?